README.md

```diff
@@ -1 +1,5 @@
-# trials-project
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please consider writing a more descriptive readme including all important information about the project so that recruiters can easily navigate you. Please follow this [template](https://github.com/microverseinc/readme-template)